app/src/main/java/com/geeksville/mesh/service/MeshService.kt

```diff
@@ -414,10 +414,17 @@ class MeshService : Service(), Logging {
 
     // given a nodeNum, return a db entry - creating if necessary
     private fun getOrCreateNodeInfo(n: Int) = nodeDBbyNodeNum.getOrPut(n) {
+        val defaultId = DataPacket.nodeNumToDefaultId(n)
+        val shortName = if (defaultId.length >= 4) {
```

Comment:
nodeNumToDefaultId() returns a fixed length of 9 characters, so this is always true and can be removed